Automattic/wp-calypso
# VideoPress onboarding v2 : steps validation

#### Proposed Changes

This PR adds validation to access different steps in the flow.
It also clears previously selected options when hitting intro instead of waiting for submit.

#### Testing Instructions

* Apply this PR
* In an incognito window
* Navigate to `calypso.localhost:3000/setup/options?flow=videopress`
* ✅ You should be redirected to intro
* Navigate to `calypso.localhost:3000/setup/chooseADomain?flow=videopress`
* ✅ You should be redirected to intro
* Navigate to `calypso.localhost:3000/setup/chooseAPlan?flow=videopress`
* ✅ You should be redirected to intro
* Log in
* Navigate to `calypso.localhost:3000/setup/chooseADomain?flow=videopress`
* ✅ You should be redirected to options
* Navigate to `calypso.localhost:3000/setup/chooseAPlan?flow=videopress`
* ✅ You should be redirected to options
* Set a site title and validate
* ✅ You should be redirected to chooseADomain
* Click on "decide later"
* ✅ You should be redirected to chooseAPlan
* Navigate to `calypso.localhost:3000/setup/intro?flow=videopress`
* Navigate to `calypso.localhost:3000/setup/options?flow=videopress`
* ✅ The options should be cleared out 

Fixes Automattic/greenhouse#1340
